tests/sdp.js

```diff
@@ -2,11 +2,43 @@
 
 const assert = require('assert');
 
-const { getPreferredCodecInfo, setCodecPreferences, setMaxAverageBitrate } = require('../lib/twilio/rtc/sdp');
+const {
+  getPreferredCodecInfo,
+  setCodecPreferences,
+  setIceAggressiveNomination,
+  setMaxAverageBitrate
+} = require('../lib/twilio/rtc/sdp');
 
 const { makeSdpWithTracks } = require('./lib/mocksdp');
 const { combinationContext } = require('./lib/util');
 
+describe('setIceAggressiveNomination', () => {
+  const SDP_ICE_LITE = 'a=ice-lite\nfoo';
+  const SDP_FULL_ICE = 'a=group\nfoo';
```

Comment:
Make it more representative of the real thing?

```suggestion
  const SDP_FULL_ICE = 'a=group\nfoo\na=ice-options:trickle-ice\n';
```